### ISSUE TITLE
Project Status

### ISSUE THREAD

**Issue by ftoledo** [cwensley/pablodraw#22]
Hi, is this project still alive?

Thanks!

**Comment by cwensley** [comment]
It sort of is, though this is more of an archive.  That being said, I have some updates coming soon such as Gtk+3, 64-bit macOS support, etc.

**Comment by ftoledo** [comment]
thanks @cwensley great news! I can test if you make a builds tags!

**Comment by cwensley** [comment]
Thanks @ftoledo, that would be awesome.  I have some new test builds available here:

- [Linux x64](https://github.com/cwensley/pablodraw/suites/1933523028/artifacts/38152839)
- [Mac x64](https://github.com/cwensley/pablodraw/suites/1933523028/artifacts/38152840) (run `xattr -c PabloDraw.app` after copying it from the dmg)
- [Windows x64](https://github.com/cwensley/pablodraw/suites/1933523028/artifacts/38152841)

These are fully self contained native executables so they no longer run or require mono/.NET.

I can provide other builds (x86, arm) if necessary.

**Comment by ftoledo** [comment]
thanks! I haven't tested it extensively yet, but it's good news and just work on debian stretch 64bits!

**Comment by cwensley** [comment]
Closing as I believe this question was answered.  I hope to get PabloDraw updated and CI (continuous integration) enabled so new releases can be easily made and published here.  I've also added a crash dialog so users can report issues with a bit more information to address some of the crashing issues.  I don't envision adding a ton of new features or anything, just bug fixes and getting things up to date.
